### ISSUE TITLE
plans to implement protocols other than ICMP and UDP?

### ISSUE THREAD

**Issue by kamihacker** [apenwarr/netselect#3]
Considering the fact that there might be environments behind firewalls or proxies, are there any plans to implement a workaround over port 80 with a simple HTTP GET request?

Thanks

CM

**Comment by TriMoon** [comment]
I don't think so, because that would need a download speed-test, which this program does not use and would be too stressful for the many mirror servers that need to be checked...
